src/users/UserItem.js

```diff
@@ -63,16 +66,17 @@ export function UserItemRaw<UserT: { user_id: UserId, email: string, full_name:
         <UserAvatarWithPresenceById
           size={48}
           userId={user.user_id}
+          isMuted={isMuted}
           onPress={onPress && handlePress}
         />
         <View style={componentStyles.textWrapper}>
           <RawLabel
             style={[componentStyles.text, isSelected && componentStyles.selectedText]}
-            text={user.full_name}
+            text={isMuted ? 'Muted User' : user.full_name}
```

Comment:
Same capitalization comment as in the other PR -- "Muted user" would be more consistent with web app.

Reply:
Ah, and: this 'Muted User' text won't be translated, even with an entry in messages_en.json. 😛

I think `RawLabel` was chosen so that people's full names don't get localized; note its jsdoc:

```
Unlike `Label` it does not translate its contents.
```

The solution is probably to do `_('Muted user')`, where `_` is `useContext(TranslationContext)`.

Reply:
Fixed :)